github/Project.py

```diff
@@ -140,6 +140,60 @@ def url(self):
         self._completeIfNotSet(self._url)
         return self._url.value
 
+    def delete(self):
+        """
+        :calls: `DELETE /projects/:project_id <https://developer.github.com/v3/projects/#delete-a-project>`_
+        :rtype: None
+        """
+        headers, data = self._requester.requestJsonAndCheck(
+            "DELETE", self.url, headers={"Accept": Consts.mediaTypeProjectsPreview}
+        )
+
```

Comment:
Other delete methods usually return True and check the status returned by GitHub, could you do the same here? You can then ``self.assertTrrue()`` in the test case.

Reply:
I checked the documentation and the result of the call. The call to [the above URL](https://developer.github.com/v3/projects/#delete-a-project), doesn't return a True or False. It just gets us `None`.

I checked the other methods as well, none of them return any data.

https://github.com/PyGithub/PyGithub/blob/b7894ea00c9993f5d129091b15ce0232da1e1aea/github/GitRef.py#L68-L75
https://github.com/PyGithub/PyGithub/blob/b7894ea00c9993f5d129091b15ce0232da1e1aea/github/Branch.py#L250-L256
https://github.com/PyGithub/PyGithub/blob/fc4eff3185af49239712dcb32779a6c51b788136/github/Migration.py#L154-L162

Let me know if there is a way around this.

Reply:
Right, I was confusing it with another method, sorry.